src/library/itunes/ituneslocalhosttoken.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <QString>
+
+static const QString kiTunesLocalhostToken =
+        "//localhost"
+#if defined(__WINDOWS__)
+        "/"
+#endif
+        ;
```

Comment:
I guess I was too clever for MSVC, then I'd prefer the verbose but still easily readable version. 
```suggestion
#if defined(__WINDOWS__)
static const QString kiTunesLocalhostToken = QStringLiteral("//localhost/");
#elif
static const QString kiTunesLocalhostToken = QStringLiteral("//localhost");
#endif
```

Reply:
Just curious, is there any advantage of explicitly calling `QStringLiteral`? We already declare it as a `QString`, so the explicit conversion feels a bit redundant.

But I'd wait for the MSVC build to complete first, I don't know whether cond-compiles are just problematic within parameter contexts or in expressions in general.

Reply:
Its somewhat redundant because it essentially saves us a single heap-allocation per process IIUC, though it still makes sense IMO.
I think the qt doc explains it best https://doc.qt.io/qt-6/qstring.html#QStringLiteral

Reply:
Ah interesting, thanks!

Reply:
> But I'd wait for the MSVC build to complete first, I don't know whether cond-compiles are just problematic within parameter contexts or in expressions in general.

QStringLiteral is a Macro, so MSVC probably doesn't like preprocessor directives within the parameters of a macro. If QStringLiteral was just a constexpr function, it would probably be fine.

Reply:
Yes, thats even better.

Reply:
Though if we're really pedantic, I'd prefer to duplicate the semicolon, its rather odd on its own line with all that whitespace
```suggestion
static const QString kiTunesLocalhostToken =
#if defined(__WINDOWS__)
        QStringLiteral("//localhost/");
#else
        QStringLiteral("//localhost");
#endif
```